apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts

```diff
@@ -214,11 +213,14 @@ const mapField = (
   field: TCreateSinglePlayerDocumentSchema['fields'][number],
   signer: TCreateSinglePlayerDocumentSchema['signer'],
 ) => {
-  const customText = match(field.type)
-    .with(FieldType.DATE, () => DateTime.now().toFormat('yyyy-MM-dd hh:mm a'))
-    .with(FieldType.EMAIL, () => signer.email)
-    .with(FieldType.NAME, () => signer.name)
-    .otherwise(() => '');
+  const customText =
```

Comment:
Just confirming that this is being reverted

Reply:
Oops! Thanks, Lucas. I had missed that one, just sent a commit to revert it :)